todo_app/app.py

```diff
@@ -1,45 +1,55 @@
 from operator import attrgetter, itemgetter
 
 from flask import Flask, render_template, request, redirect, url_for
+from datetime import datetime
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item, get_item, save_item, remove_item
+from todo_app.data.trello_items import get_all_items, create_card, delete_card, complete_item, undo_complete, edit_desc, edit_due_date
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = sorted(get_items(), key=lambda i: i['status'], reverse=True)
-
+    items = sorted(get_all_items(), key=lambda i: i.status, reverse=True)
     return render_template('index.html', items=items)
 
 
 @app.route('/add', methods=['POST'])
 def addItem():
-        title = request.form.get('title')
-        add_item(title)
-        return redirect(url_for('index'))
+    create_card(request.form.get('title'))
+    return redirect(url_for('index'))
 
 
 @app.route('/complete', methods=['POST'])
 def completeItem():
-        item = get_item(request.form['completeBtn'])
-        item['status'] = "Completed"
-        save_item(item)
-        return redirect(url_for('index'))
+    complete_item(request.form['completeBtn'])
```

Comment:
What does `request.form['completeBtn']` represent? Is it the task title, id or something else?

I think it would be better to declare a variable e.g. `item_id` with a helpful name.